### PR TITLE
Remove The Justice Department

### PR DESCRIPTION
**Pull request by MilonPL**
## About the PR

Removes Clerk, Chief Justice and the Justice department.
Move Attorney to Service.
Added SOP and Space Law to the guidebook.
Adjusts security roles and loadouts, moved prosecutor to sec. They get lawyer + sec access. 
Errrm powergaming?? Just roleban them™

## Why / Balance

I'm tired of all the whining.
Justice was a mistake. It was a good idea on paper, for someone, for sure. But we don't need a whole ass legal system or impartial prosecutors. Prosec is a part of security now, they can sit at the front desk and take criminal reports (WHICH EVERYONE ALREADY DOES). If you want to sue sec, go yell at a lawyer. No one plays CJ anyways, you can just ask a head of staff to do a trial. I used to think that trials suck, but turns out that we had a *good* thing that someone tried improving, and it only went downhill from there. You can't fix this, and it took me too long to realize that.

Justice was the long anticipated solution to all the problems with trials, but it has only brought more issues than it fixed. The entire SOP revision was centered around justice, yet there's *tons* of problems with the SOP. Sitting in a 2 by 2 cell for an hour isn't engaging roleplay, and it's understandable how people just SSD when being caught. I don't think it needs an entire department - let Warden do the parole hearings for their perma. Let the prosecutor oversee sentencing or take criminal records.

## Technical details

"We can't revert justice" my ass. 
deltamigrations.yml

## Media

image
image

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- remove: Removed the justice department.
- tweak: Moved Prosecutor and Attorney to Security and Service respectively.
- add: Added SOP and Space Law to the guidebook.

### CODE REVIEW

**Comment by flowerfall-system** [comment]
We support this with every ounce of fervor we can muster. We're not just regular Justice haters - we're PROFESSIONAL Justice haters. We wrote a fuggin' Google Doc about it. Read: https://docs.google.com/document/d/1HamegAhOSXSUJKF7tzf8qxzOnBxvRfKjvswRXv4Wx-g/edit?usp=sharing

**Comment by Lyndomen** [comment]
net removal of content.jpg 
:godo:

**Comment by MilonPL** [comment]
> net removal of content.jpg :godo:

ok but the content sucks so

**Comment by therealDLondon** [comment]
I will fight against this PR, not only is it written out of spite, evident from the language, but I feel there are better solutions than just gutting the entire department because "Muh justice bad"
We have new SOP, lets let that sink in for a while and see how it goes

**Comment by Lyndomen** [comment]
ok lets remove ghost soft toys, nukies, cloning, psionics, AMEs, TEGs, revenants, expeditions, warops, dragon objectives, and im sure i can thing of other stuff first, cause all that has sucked for longer

**Comment by MilonPL** [comment]
> ok lets remove ghost soft toys, nukies, cloning, psionics, AMEs, TEGs, revenants, expeditions, warops, dragon objectives, and im sure i can thing of other stuff first, cause all that has sucked for longer

So far the answer to reverting justice has always been "we can't". Well, here's that option. Let's see what people think.

**Comment by LioValkyria** [comment]
While the concept of Justice Department is good, in all the time I've played so far I've never seen it get utilized.
It simply takes too long for a single trial to happen and too many cogs are needed for it to happen as it is currently.
Is it really a net removal of content if it's speeding up parts of said content to be handled?

**Comment by MilonPL** [comment]
Pretty much all of the items that the justice department PR added like trial timers are staying, the only thing getting removed is access and jobs. The clothes are also staying, they can be pre-mapped or added to loadouts later on.

**Comment by leonardo-dabepis** [comment]
I am admittedly biased as the person who originally created the Justice Department PR (though I my contributions are basically only the sprites and some yaml, I had passed off the project to other people when I left and they all did the vast majority of the work for this department) but I would rather not see it removed just because some people have a hard time with it.
Having recently started playing as a pseudo CJ main and with a stupid amount of hours in lawyer, I think it's a VAST improvement from the system we had before. Trials actually happen, sec can actually do security while the people they catch get processed instead of having half the sec department holed up in the court room. Before JD, there was hardly anything for lawyers to do and trials never happened because no one in security or command wanted to put up with that shit. Lawyer was essentially clown in a suit and tie, which is not really fun for what's supposed to be an MRP server.
If anything, we should just merge prosecutor and attorney into one role. That's how I originally intended it before I handed the project over. I think the one extra role is a bit too much and adds a bit of confusion.

**Comment by Spielern** [comment]
The addition of Justice has made security a complete headache to play. Syndicate rounds are so stressful, more so from having to negotiate with the prosecutor rather than the actual antagonists. It's just a headache. I cannot deny that roleplay is better FOR justice and that it is an improvement in that/adjacent regards, but there are glaring flaws with the department as a whole. As much as I love seeing trials actually happen more frequently, I think the annoyances and grievances from MANY community members far outweigh the benefits. I've even personally vowed to stop playing security until justice is fixed/removed, or space law is better. Additional departments should not harm existing ones or make them less fun to play.

**Comment by beck-thompson** [comment]
> Trials actually happen

At least from what I see, they don't! I've tried to get a trial going so many times but it is so difficult to actually do it properly, it will almost never finish before evac arrives or something goes wrong. Not to mention, forcing the defendant to wait in a holding cell for (In the very best case) 30 minutes is really asking a lot. From what I see, the lawyers hardest job is to keep the defendant from SSDing before the trail even begins (I've had them beg in LOOC chat for me to stay).

In concept, its a super fun idea, but in reality it falls short. I used to be a laywer main, but justice really took out all the fun out of it.

**Comment by Stop-Signs** [comment]
I think the thing that most justice defenders miss is that there's never justice on those roles anyway. CJ and clerk players are almost entirely non-existent, lawyer and prosecutor are the role I most see joining the round and going ssd after 10 minutes. sitting in sec waiting for a trial or legalities to be needed ISNT engaging gameplay. we aren't removing content because there _WAS_ no content, just this husk of a good idea that didn't end up working out. SOP is something that still isn't fixed after 5 months, we have gotten a single hotfix after about 3 months of dealing with lawyers loopholes and it simply makes sec unfun to play. Having to deal with the syndie who's screaming shitsec cause you caught them is headache enough, nobody benefits from yet another person yelling at sec to "do their jobs right". and none of SOP really matters due to LOTS of staff inconsistency regarding it. justice was a failure on multiple levels, it really did seem like a great idea, but its time to come to the reality that its the least liked/played department, and for good reason

**Comment by FieldCommand** [comment]
I don't particularly care for the justice department and have my own thoughts about it, and it is slowing the game down significantly and removing a lot of joy. The rule and SOP changes made it also very murky to navigate the rules and SOP. I get the idea of gray areas and loopholes, but ultimately it contributed to everything turning into a slog and making security a very unenjoyable experience. And DeltaV is like the only server I know that has this department. I think the department has to go, but I think there can be a middle ground to salvage the essence of the idea while reverting back to the old ways.

So here is the idea.
Atterny: they goes back into the service department and give back security access like they used to (this also has to reflect on the hop)
Prosecuter: gets absorbed into security. In essence, they are a full fledged security officer, but in general, they are there to prepare and represent the state when court trials are set. But are there to advise prisoners, guide, and heap teach the laws, sop, and things of that sort? (Think of them as a security officer with a desk job instead of them being out in the hallways, They can be should there be a need though.)
Clerk: will get axed entirely. some of the gear and items they have can go to the last role to cover
Chief Justice: This role needs a name change. and would not be the head or have a department, but they could be absorbed into the command structure. I don't remember what PR it was in, but there was talk of having the administrator, sort of having an assistant to command. The essence of the CJ role could fall on this role that was never implemented. Giving command the assistance I think they need and having them fill important rolls like judge when needed.

Why am I thinking this? Just having a department like justice, just crates division, regardless of how well intentioned and cooperative they are supposed to be, this is just human nature. But there needs to be roles that can fill the 3 parts of a court when they are needed. Command is usually just to busy keeping their own departement together to have time to worry about holding court, so the judge role can fall to the administrator. Sec is always busy, so having a dedicated security officer with a dedicated desk job for this could help with out the need for friction it makes. The attorney does not really change all that much other than just becoming service again. and Clerk is just a redundant role. In fact, the administrator could be called a clerk, I think.

my thought, hope they make sense.

**Comment by Tmanzxd** [comment]
Add social gameplay feature to encourage players to interact more, make it reliant on the playerbase

Said gameplay feature makes department that has power over Security

Playerbase proceeds to abuse and neglect said system provided by the department, just like Security

**Comment by SolStar2** [comment]
However, I would love it if the guidebook was in a separate PR because I want that regardless of the decision here.

**Comment by Stop-Signs** [comment]
Im all for this PR. Sec has been a drag and massive headache causer recently due to the faults of the SOP as well as relying on the least played department to get anything done. Also to @therealDLondon , we don't have new SOP, we have a hotfix that changed 2 sentences in order to fix MASSIVE loopholes, one of which you yourself caused when you deemed that "passengers aren't crew".

**Comment by HTMLSystem** [comment]
I think this PR is rash, and useless. This won't fix any problems that exist now and just make things worse, not to mention the load this will present for map makers. 
Just look at the convo in server-grievences. The one person arguing for the removal of justice the most only brings up problems that are player issues. I.E. Justice members going SSD and command members refusing to be a judge/not knowing trial procedure. In my opinion all problems people say are "justice problems" have always existed, and removing justice won't do anything but make it worse.

**Comment by Junerisms** [comment]
I've been spending a decent amount of time as a Lawyer recently, so when I noticed this I thought that maybe I should give my own input on the matter, as I actually wished to learn the ins and outs of justice so then I could confidently apply for the whitelisted CJ role.


In all of my time playing as an Attorney (didn't start as a prosecutor yet) I will admit that usually in most cases, the courtroom is as vacant as the Sahara desert. This is mainly just due to the fact that I might be the one and only person even IN the justice department. Sometimes it could be another attorney or a prosecutor. Rarely; a clerk.  I have never seen a Chief Justice throughout all of my time playing on Delta-V. I thought I could switch that up, up until this GitHub was made at least. There was just one Chief Justice, and it was someone who was anointed for that role, AND this was before the Space Laws were updated, so they announced every single possession was entirely legal. You could even have severe explosives in your hands as long as you didn't use them. 


I've been focusing a lot more on the justice department as of late ever since the Laws were updated and made controlled items contraband. It seemed like Lawyers now officially had a lot more clients to defend, and a lot more cases were now being pushed off all these contraband arrests and whatnot. I personally never found myself to get in SEC'S way at all, I found trying to get some extra leniency or some time off to be quite fun to negotiate with, whether it be the Warden or the Arresting Officer in general. While I do think Court is relatively useless, I've almost had a full fun court session myself, however it was dragged on for far too long with people preparing before it could even start; right at the opening statement the Evac shuttle docked so the court was instantly shut down. Trials are certainly something that should be looked at rather than outright permanently removed. I've never considered forcing sec to have a 'desk job' at court. It just seems like Court RP is something among the HRP servers not short time MRP like Delta-V and others. From my gameplay experience (ofcourse this can be taken with a grain of salt!) I'd say about HALF of the Delta-V rounds are basically over when the first automatic shuttle is called, Criminals could be getting Trial much-much later! It's hard to get caught the first 10 minutes into a shift to get a court case done. All the Syndicates or Traitors or Hooligans usually get pulled in at the middle or the end of the shift, where there's already enough chaos that evidently: NO ONE GOT TIME FOR COURT!!!.


In conclusion; PLEAAASEEE don't _entirely_ remove the Justice Department, I think it might just need a rework-- or two..
I would still absolutely love to be a Chief Justice, not only for the role but for the insane drip too. I'm definitely considered a very new player to Delta-V, a little over 66 hours at the current time of writing and I've latched onto Justice, The RP I've been getting from it is absolutely too good to turn down! Filing lawsuits, threatening lawsuits, gathering statements  FOR said lawsuits.. It's been a freaking blast, and I just wish I could've seen the Justice Department in it's full glory. (If/When there was a time people actually took all of the Justice Department roles) I'm the Lawyer for YOU!!!

**Comment by beck-thompson** [comment]
> Just look at the convo in server-grievences. The one person arguing for the removal of justice the most only brings up problems that are player issues. 

To be clear, this is not what I was arguing. The conversation was very long so I understand people getting confused so I'll go over it once more here in github!

1.) I don't think justice should be removed. I think it needs changes and leaving it in its current state and hoping people follow SOP better is (In my opinion) not a viable solution for fixing its issues.

2.) If justice went smoothly and everyone followed SOP fully and did their jobs competently then justice works amazing. I truly mean this! The system is well thought out and when it works, it works well. I also have no issue with anything not relating to trials (Warrents etc..). For the most part, I think those are done as well as they can be. However, I really think the *essence* of justice are the trials so that's why I'm focused on them.

3.) I personally think that getting trials actually finished is important \*. As of right now, its very time consuming to properly get all the evidence, witnesses and actually doing the trial itself in a fully fair and SOP approved manner (This is also still assuming everyone knows SOP fully, doesn't go SSD etc...). This leads to many trials never actually being started, or being started too late and evac getting called in the middle. Now, to be clear, these trails are "fair" (As fair as you can get in a game like ss14 😆) but I don't think that's worth the tradeoff. I'd rather have an "unfair" trial that actually gets completed instead of rotting in my cell just waiting and never even having a shot at getting out.

I was also told that if I can't think of any solutions I can't complain which is fair. Here are some ideas that I think could help solve the issues with justice:

**1.) Remove the clerk**
  I've only seen one clerk (They were really good!) and they were kind of helpful, but the trial could easily have gone on without them. Right now it just adds one more roll that can take a spot from more important jobs / slow things down and should be removed to streamline the trial process.

**2.) Change SOP so anyone can be judge (If both parties agree)**
  Command is usually busy and don't want to be a judge for a trial they know nothing about. I think this is reasonable and also expected! I think if both parties can agree on a party, then they can be the judge. This goes back to "fairness", yes this will be less "fair" than if you found an unbiased head, but it will significantly speed up the process of finding a judge (This seems to be a big blocker from what I see). I think its worth the tradeoff.

**3.) Law book / portal in cells that clearly outlines the trial procedure.**
  Add a mapped terminal in each cell that clearly outlines the process of a trial and what steps should be taken. This should be short, and to the point. Now, if an officer says "I can't charge you, only CJ can", you can point to an in game item that clearly states otherwise. This will hopefully stop a lot of the "Its not my job" arguments when it is, in fact, your job to do something.

**4.) Have other gameplay for justice.**
  This is a hard one and I don't know specifically what should be done. Right now, its not even guaranteed to have even a single justice player in any given shift. This is compounded by the fact that they seem to go SDD quite frequently if nothing is going on. If there was some other sort of gameplay mechanic that they could interact with that might help bring more players and keep them playing. What specifically? I don't know, and that's hard to answer. I was thinking they could maybe do audits or something but that could lead to even more issues... If anyone has suggestions, post them for sure!

As a final note, I don't think justice ruins the game or anything. I personally think trials are *slightly* worse than before justice but they weren't exactly amazing before so it doesn't matter that much. Justice can be fixed, its just a matter of figuring out what works best!
 
\* Also, for everyone asking me what, specifically, my issue with justice is that's it! Trials take too long are rarely started, and almost never completed.

**Comment by deltanedas** [comment]
removing justice is just not the right course of action for the many many reasons, including mappers having to downsize court.

clerk can die though its a completely boring role that is only ever played by people farming hours for CJ, it has barely any purpose.

as beck said, justice should have incentive to play without antags so on survival theyre not just being tiders from nothing to do and thus absent like almost every round

trials could also have an addition to sop that they cant have evac called during them, ive personally had people let auto shuttle cut off my trial because they dont care enough to actually give a verdict (the entire point of a trial) to click the button

**Comment by LadyDanger2743** [comment]
As I'm sure others have said here, this PR is wholly meaningless. It will not actually fix any of the issues with Justice- or, to narrow down onto the point, with TRIALS. Justice has become a personification of the trial process, yes, but its removal will not cure us of the ills of trial procedure. Pretending that this would "fix" anything is very arrogant. 

SOP needs fixes. Space Law needs fixes. Hell, Justice role times and job counts might need fixes! But killing Justice won't actually fix the underlying issue, it's just a bunch of performative nonsense.

**Comment by Monotheonist** [comment]
Okay, but like, and I'm just making an observation:
The community has always been really split on this (no fucking shit, Mono!!) 
It's part of the reason why I still haven't joined the discord, because ostensibly as a contributor who wants to make a change and as a player who always hears about things playing out weirdly with this debate I don't like hearing this.
I think it's been the source of like 3 separate maint breakdowns over time, having this debate, because a chunk of the community wants justice to be a certain way with certain specific roles omitted and a other, really big group wants justice to be wholly gone because of how it goes into play with Security. No one wants to act, and any action becomes a point of controversy because the action is usually the removal.
It sucks. What it seems to look like is people can badger others into making PRs like this, and it can become a point of stress. 

So this comes from the heart:

To all of you Justice dislikers, maybe instead of bashing the department entirely put in the effort to debate and rewrite space law and address the core problems of SoP and Space Law. Address your grievances with it and make it part of the eternal debate on what to do with justice. You can do it. Beck even mentions some really good starting points related to Justice at a surface level on how to make immediate improvements that I can get behind as someone who's played Justice. If you get angry about it, take a break and calm yourself down and remind yourself **this is about a law department in a video game.**
Hatred of this department and a lack of consensus on what to do with it makes no one want to touch it, and when a maintainer or someone else wants to do something about the surmounting community feedback, it's quite hard to approach it with a cool head and it often ends in this. I'm really just echoing the sentiment of the last three comments here when I say this, there are other ways to deal with this other than just slinging hatred at it.
That's my cringe PSA, I'll probably delete this tomorrow. Over and out.
